api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -1,10 +1,20 @@
 const _ = require('lodash')
 const BigNumber = require('bignumber.js')
-const { fixDecimalsAndRoundUp } = require('../../common/numbers.js')
+const {
+  fixDecimalsAndRoundUp,
+  toViewDenom
+} = require('../../common/numbers.js')
 const { lunieMessageTypes } = require('../../lib/message-types')
 
 const CHAIN_TO_VIEW_COMMISSION_CONVERSION_FACTOR = 1e-9
 
+const proposalTypeEnum = {
```

Comment:
in cosmosV2 this is called `proposalTypeEnumDictionary` ... can we make them the same?

Reply:
They are two different things, that is why they are called differently. In the cosmos source it is a real dictionary, converting from cosmos types to Lunie proposal types, while here it is just an enum